Source/Optional.swift

```diff
@@ -84,3 +84,10 @@ public extension Optional {
         return f.flatMap { self.map($0) }
     }
 }
+
+func foo(bar: String!) {
+  let x: String! = "howdy"
+  print("hello world: \(bar!)")
+}
+
+let x :String = "break it"
```

Comment:
Colon Violation: When specifying a type, always associate the colon with the identifier<br>Name Format Violation: Variable name should be between 3 and 40 characters in length: 'x'